src/Snppts/Snppts.csproj

```diff
@@ -23,7 +23,5 @@
     <Folder Include="Views\Shared\Components\PopularCategories\" />
     <Folder Include="Views\Shared\Components\AllCategories\" />
     <Folder Include="Views\Shared\Components\RandomSnippets\" />
-    <Folder Include="Snippets\" />

```

Comment:
It's probably nothing and was done by Visual Studio automatically but could you undo the changes here please? Just to make sure we don't break anything.

Thanks!

Reply:
It's ready ;)